homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -171,3 +179,13 @@ def native_value(self) -> StateType:
         """Return the state."""
         state = getattr(self._vehicle.status, self.entity_description.key)
         return cast(StateType, self.entity_description.value(state, self.hass))
+
+    @property
+    def extra_state_attributes(self) -> Mapping[str, Any] | None:
```

Comment:
We don't normally want any extra state attributes in sensors. They should be separate sensor entities. Please list what values these attributes are.

Reply:
Moved to #60942 and added an additional sensor